### ISSUE TITLE
Problem about interface background alpha

### ISSUE THREAD

**Issue by H-F-M** [tateisu/SubwayTooter#21]
When you Copy/Past a value, the 'coma' (in french we don't use 'dot' for decimal separation) disappear ex: "0,3000" come to "03000" when pasted. If i use the ext-keyboard, the 'coma' don't write down but i can put a 'dot' instead and it's ok with int-keybord, no choice of 'coma' only 'dot' is present.

An idea ?

After reflection, i think it is relate to input method used, with int-keyboard, only numeric function come, not the full keyboard interface. Maybe a way ?

**Comment by tateisu** [comment]
seems it came from inputType="numberSign" attribute for EditText UI element. 
http://stackoverflow.com/questions/3821539/decimal-separator-comma-with-numberdecimal-inputtype-in-edittext

maybe adding  android:digits="0123456789.," attribute  in EditText  in app/main/res/layout/act_column_customize.xml  to fix it.

**Comment by H-F-M** [comment]
ok seems to be the solution, cause when i change the 'coma' by 'dot' before copying, and i past it, it's ok
 (it past with 'dot' but work well).

**Comment by tateisu** [comment]
also, Float.parseFloat(String) does not support locale.
it need to be changed using NumberFormat.parse()

```
@Override public void afterTextChanged( Editable s ){
				if( loading_busy ) return;
				try{
					NumberFormat format = NumberFormat.getInstance( Locale.getDefault() );
					Number number = format.parse( etAlpha.getText().toString() );
					float f = number.floatValue();
					if( ! Float.isNaN( f ) ){
						if( f < 0f ) f = 0f;
						if( f > 1f ) f = 1f;
						column.column_bg_image_alpha = f;
						ivColumnBackground.setAlpha( column.column_bg_image_alpha );
						sbColumnBackgroundAlpha.setProgress( (int) ( 0.5f + f * PROGRESS_MAX ) );
					}
				}catch( Throwable ex ){
					log.e( ex, "alpha parse failed." );
				}
			}

```

I've commit to [master b8286aa] fix alpha value parser

**Comment by H-F-M** [comment]
I understand briefly, I use Delphi for my small developments, it is Pascal not "C" or java :p 
I try to be just as precise as possible in my explanations to make your task easier.

**Comment by tateisu** [comment]
fixed on v0.5.7.

**Comment by H-F-M** [comment]
Right you're the best :) !
